modules/gcp/gke/main.tf

```diff
@@ -152,6 +149,14 @@ resource "google_project_iam_member" "service_account-roles" {
   member  = "serviceAccount:${google_service_account.gke_service_account.email}"
 }
 
```

Comment:
Please put a comment here about why this is needed.

Reply:
Done.

Reply:
This comment doesn't match what you're actually doing. How does `roles/storage.objectViewer` provide access to stackdriver logging, trace, etc?  

The real issue is that `roles/storage.objectViewer` doesn't seem obvious that it grants access to pull from GCR. You managed to explain the real purpose for this role in the PR but yet you put something completely unrelated in the actual comment. This is worse than not having something at all.

Reply:
The comment is not related to `roles/storage.objectViwer` but the previous resource declaration where `"roles/logging.logWriter"`, `"roles/monitoring.editor"`, `"roles/cloudtrace.agent"` are assigned to the gke SA. I'll add a comment to the resource `resource "google_storage_bucket_iam_binding" "gke_cluster_members"` that is the one you are talking about.